### DISCUSSION TITLE
Should Renovate generate PRs for pyproject.toml build-system section?

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#17625] [Request Help] [answered]
Here is an example:
`[build-system]`
`requires = ["setuptools==63.2.0"]`
`build-backend = "setuptools.build_meta"`

I would expected renovate to locate the setuptools pinned version and raise a PR. 
Does it make sense?

**Comment by PhilipAbed** [comment, marked as answer]
no we dont support that yet

**Comment by rarkins** [comment]
Related to https://github.com/renovatebot/renovate/issues/10187?

**Reply by lmilbaum** [reply]
> Related to #10187?

Makes sense
